caesar_cipher/lib/cipher.rb

```diff
@@ -0,0 +1,29 @@
+class Cipher
+
+  def initialize(rot_key, raw_message)
+    @rot_key = rot_key
+    @raw_message = raw_message
+  end
+
+  def encrypt
+
+    encrypted_message = ''
+    @raw_message.each_char do |character|
+      character = character.ord
+      if character.between?(65, 90)
+        character += @rot_key
+        character -= 26 if character > 90
+        encrypted_message << character.chr
```

Comment:
@zzokki81 you have three identical lines like this one. Can you move them out of the `if` block to be executed after it?